Standards/scs-0004-v1-achieving-certification.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Regulations for achieving SCS-compatible certification
+type: Procedural
+status: Draft
+track: Global
+---
+
+## Introduction
+
+The Sovereign Cloud Stack (SCS) issues certificates with various scopes, among them _SCS-compatible IaaS_ (infrastructure as a service) and _SCS-compatible KaaS_ (Kubernetes as a service).
+
+This document details how a cloud service provider (henceforth also called operator) can attain such a certificate for one of their clouds.
+
+## Motivation
+
+As operator, I want to obtain a certificate with the scope SCS-compatible IaaS or SCS-compatible KaaS.
+
+## Regulations
+
+1. Each certificate issued pertains to a given cloud, a given scope, and a given version of that scope with a fixed expiry date. The certificate is only valid for that cloud and for the time frame that ends on that expiry date.
+
+2. The operator MUST include the official [SCS compliance test suite](https://github.com/SovereignCloudStack/standards/tree/main/Tests) (which does not require admin privileges) in their continuous test infrastructure (e.g., Zuul). The tests MUST be run at given intervals, depending on their resource-usage classification:
+
+    - _light_: at least nightly,
+    - _medium_: at least weekly,
+    - _heavy_: at least monthly.
+
+   For public clouds, it is recommended to offer the SCS project access to the infrastructure so the test suite runs can be triggered continuously by the SCS team.
+
+   Alternatively, and for non-public clouds, the results (log files) MUST be submitted to SCS by a mechanism of SCS' choice and need to be reproduced again on request by SCS.
+
+   <!-- Initially this will probably be eMail -->
+
+3. If the desired certificate requires manual checks, then the operator MUST offer the SCS project suitable access. Manual checks MUST be repeated once every 3 months.
+
+4. Details on the standards achieved, as well as the current state and the history of all test and check results will be displayed on a public webpage (henceforth, _certificate status page_) owned by SCS.
```

Comment:
As we publish results of SCS conformance tests, we need the permission of CSP to do so in order to prevent any legally conficrts. I do not know, if this should be part of "Regulations for achieving SCS-compatible certification". However, we should keep this in mind and add an appropriate statement to SCS certificate's terms and conditions.

Reply:
My understanding is that any CSP objecting to this kind of publication should NOT apply for certification. Nobody is being forced.

Reply:
I guess we need to write a little "SCS certification terms & conditions document" that states obligations and conditions. I don't to sign individual contracts, so it's a T&C thing. (In German: Allgemeine Geschäftsbedingungen.)